src/services/identity/agent.ts

```diff
@@ -240,22 +239,23 @@ export class Veramo {
   }
 
   async verifyCredential(agent: VeramoAgent, credential: string | VerifiableCredential, verificationOptions: VerificationOptions = {}): Promise<IVerifyResult> {
-    const decodedCredential = typeof credential === 'string' ? decodeJWT(credential) as unknown as VerifiableCredential : credential
     let result: IVerifyResult
     if(verificationOptions.verifyStatus) {
         result = await agent.cheqdVerifyCredential({
             credential: credential as VerifiableCredential,
             fetchList: true,
             verificationArgs: {
-                ...verificationOptions,
-                fetchRemoteContexts: verificationOptions.fetchRemoteContexts || decodedCredential.proof.jws
+                ...verificationOptions
             }
         } as ICheqdVerifyCredentialWithStatusList2021Args)
     } else {
         result = await agent.verifyCredential({
             credential, 
             ...verificationOptions,
-            fetchRemoteContexts: verificationOptions.fetchRemoteContexts || decodedCredential.proof.jws
+            policies: {
+                ...verificationOptions.policies,
+                credentialStatus: false,
```

Comment:
why did we add here `credentialStatus: false`? Does it mean that we want to make such field not mandatory?

Reply:
credentialStatus true is handled in the above condition which call's did-provider-cheqd cheqdVerifyCredential

the native verifyCredential doesn't support our status. if verifyStatus is set to false in request, it move to this condition